src/queue/display/index.ts

```diff
@@ -10,7 +10,9 @@ export enum SCREENS {
 }
 
 let frameEntity: Entity
-let positionActive: TransformType
+// TODO: use for now for moving the player outside the game area.
+// it should use the config.queueDisplay once we move this logic there
+export let positionActive: TransformType
```

Comment:
Just to be sure, positionActive on queueDisplay is used for setting position of the display device. 
Did you check if moving the player to this coordinate doesn't get it trap in a collider?

Reply:
no, we dont have that method. Maybe i can move it 1 meter backwards, so we dont need to pass an extra config param (we have too many already)